src/Microsoft.Framework.Logging.Debug/project.json

```diff
@@ -4,6 +4,7 @@
         "type": "git",
         "url": "git://github.com/aspnet/logging"
     },
+    "description": "Debug logger implementation.",
```

Comment:
Can these descriptions say something a bit more useful? `Logger implementation that writes to debug output."